src/osd/PGLog.h

```diff
@@ -1112,19 +1112,27 @@ struct PGLog : DoutPrefixProvider {
 			   << "," << info.last_update << "]" << dendl;
 
 	set<hobject_t, hobject_t::BitwiseComparator> did;
+	set<hobject_t, hobject_t::BitwiseComparator> del;
 	set<hobject_t, hobject_t::BitwiseComparator> checked;
 	for (list<pg_log_entry_t>::reverse_iterator i = log.log.rbegin();
 	     i != log.log.rend();
 	     ++i) {
-	  if (!debug_verify_stored_missing && i->version <= info.last_complete) break;
 	  if (cmp(i->soid, info.last_backfill, info.last_backfill_bitwise) > 0)
 	    continue;
 	  if (i->is_error())
 	    continue;
+	  // merge clean_regions in pg_log_entry_t if needed
```

Comment:
I don't think that's true.  The recent missing_set changes *always* write down the missing items right?